packages/a11y-base/src/field-aria-controller.js

```diff
@@ -97,7 +97,7 @@ export class FieldAriaController {
    * @private
    */
   __setLabelIdToAriaAttribute(labelId, oldLabelId) {
-    this.__setAriaAttributeId('aria-labelledby', labelId, oldLabelId);
+    setAriaIDReference(this.__target, 'aria-labelledby', { newId: labelId, oldId: oldLabelId, fromUser: false });
```

Comment:
We could omit `fromUser: false` from here and assume it's set to `false` by default.

Reply:
Done.